portfolio/src/main/webapp/script.js

```diff
@@ -26,7 +26,11 @@ function promptRandomQuote() {
   // Pick a random quote.
     const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
-  // Show message.
-  // TO DO: make message a pop up
-    alert(quote)
+  // Return quote as string to be used elsewhere.
+    return quote;
 }
+
+function popupMessage() {
+    var quoteToDisplay = pickRandomQuote();
```

Comment:
nit - per [style guide](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let) prefer `const` over `var`.